src/NuGet.Clients/NuGet.VisualStudio.Common/PackageLevelType.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.VisualStudio
+{
+    public enum PackageLevelType
```

Comment:
I would remove `Type` suffix, since `enum` type suggest a finite, categorical variable.

```suggestion
    public enum PackageLevel
```